.github/workflows/zz_generated.gitleaks.yaml

```diff
@@ -0,0 +1,17 @@
+# DO NOT EDIT. Generated with:
+#
+#    devctl gen workflows
+#
+name: gitleaks
+
+on: [push,pull_request]
+
+jobs:
+  gitleaks:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: '0'
+    - name: gitleaks-action
+      uses: zricethezav/gitleaks-action@v1.1.4
```

Comment:
Do we need to pass any configuration to this or just run it with default settings?

Reply:
Just the defaults. If all goes well we won't need to customize it :crossed_fingers:

Reply:
Cool, let's see then.